com.lablicate.OpenChrom.metainfo.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <id>com.lablicate.OpenChrom</id>
+  <name>OpenChrom</name>
+  <summary>Chromatography Data System</summary>
+  <developer_name>Lablicate GmbH</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>EPL-1.0 AND LicenseRef-proprietary=https://lablicate.com/about/terms</project_license>
```

Comment:
Is redistribution allowed under the license? Did you contact the upstream?

Reply:
Yes, redistribution is allowed. Everything that is proprietary can be downloaded at runtime or is part of the internal distribution mechanism. The AppStream metadata file explains the concept. Note: this does not apply to older versions.

Reply:
Where is this written and which part is under the proprietary license?

Appstream doesn't matter.

Unless explicitly written that redistribution is allowed, proprietary licensed stuff needs to use extra data

Reply:
It was simpler to remove all proprietary bits and download them at runtime inside the application instead. https://github.com/OpenChrom/openchrom/pull/481